ml-agents/mlagents/trainers/buffer.py

```diff
@@ -255,6 +255,35 @@ def truncate(self, max_length: int, sequence_length: int = 1) -> None:
             for _key in self.keys():
                 self[_key] = self[_key][current_length - max_length :]
 
+    def resequence_and_append(
+        self,
+        target_buffer: "AgentBuffer",
```

Comment:
Why do we have this magic string here ? Will it cause problems if a user names a brain AgentBuffer?

Reply:
This isn't a magic string - it's a type annotation :P -> https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html#miscellaneous